.cirrus.yml

```diff
@@ -89,10 +71,38 @@ linux_task_template: &LINUX_TASK_TEMPLATE
       - sha256sum ${CIRRUS_WORKING_DIR}/requirements/ci/py$(echo ${PY_VER} | tr -d ".").yml
 
 
+#
+# YAML alias for compute credits
+#
+compute_credits_template: &CREDITS_TEMPLATE
+  # Only use credits for non-DRAFT pull-requests to SciTools/iris master branch by collaborators
+  use_compute_credits: $CIRRUS_REPO_FULL_NAME == 'SciTools/iris' && $CIRRUS_USER_COLLABORATOR == 'true' && $CIRRUS_PR_DRAFT == 'false' && $CIRRUS_BASE_BRANCH == 'master' && $CIRRUS_PR != ''
```

Comment:
@tkknight  Yup, true. But in the process of working out the above correct `CIRRUS_` environment variables, I could easily turn the compute credits on and off... so I believe that there is as much fine control as we may need here.

As they are free credits we can just use them up and monitor their usage and experiment a little to understand how we might want to use them (and control their use) when and if we actually commit to paying for compute credits

Reply:
This seems like a pretty low risk change, especially if we get booted down to the same community tier we're currently on when credits run out.  it would be good to get this in to gather data on how many credits a single suite run will take.